.gitmodules

```diff
@@ -5,3 +5,6 @@
 [submodule "extern/juce"]
 	path = extern/juce
 	url = https://github.com/juce-framework/JUCE.git
+[submodule "extern/spdlog"]
```

Comment:
Popular library used for logging. I believe it will be useful for debug/error logs in tests or some parts of production code

Reply:
```sh
[2024-05-22 00:15:18.000] [debug] [OS ENV] VIRTUAL_ENV: /home/mszawerd/venom/venom/cmake-build-debug/tests/vst3/venv
[2024-05-22 00:15:18.000] [debug] [OS ENV] PYTHON_HOME: Not set
[2024-05-22 00:15:18.000] [debug] Successfully preloaded shared library: /usr/lib/x86_64-linux-gnu/libpython3.11.so
[2024-05-22 00:15:18.000] [debug] [PY ENV] CWD: /home/mszawerd
[2024-05-22 00:15:18.000] [debug] [PY ENV] PYEXEC: /home/mszawerd/venom/venom/cmake-build-debug/tests/vst3/venv/bin/python3
[2024-05-22 00:15:18.000] [debug] [PY ENV] PY_VER: 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0]
[2024-05-22 00:15:18.000] [debug] [PY ENV] SYS_PATH: ['', '/usr/lib/python311.zip', '/usr/lib/python3.11', '/usr/lib/python3.11/lib-dynload', '/home/mszawerd/venom/venom/cmake-build-debug/tests/vst3/venv/lib/python3.11/site-packages']
```